components/microsoft_onedrive/actions/download-file/download-file.mjs

```diff
@@ -0,0 +1,48 @@
+import { axios } from "@pipedream/platform";
+import fs from "fs";
+import onedrive from "../../microsoft_onedrive.app.mjs";
+
+export default {
+  name: "Download File",
+  description: "Download a file stored in OneDrive. [See the documentation](https://learn.microsoft.com/en-us/onedrive/developer/rest-api/api/driveitem_get_content?view=odsp-graph-online)",
+  key: "microsoft_onedrive-download-file",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    onedrive,
+    filePath: {
```

Comment:
If possible, use async options here, to enhance the user experience. (see OneDrive documentation)[https://learn.microsoft.com/en-us/onedrive/developer/rest-api/api/driveitem_search?view=odsp-graph-online]

Reply:
Excellent suggestion, thanks - I've implemented this as an alternative to specifying the file path, rather than replacing it entirely.